src/config/index.ts

```diff
@@ -4,6 +4,13 @@ config();
 export const baseUrl =
   process.env.BASE_URL ?? process.env.NEXT_PUBLIC_BASE_URL ?? "";
 
+export const twak_property_id = process.env.NEXT_PUBLIC_TWAK_PROPERTY_ID;
```

Comment:
NEXT_PUBLIC makes the secret public in the document source when a user inspects the browser for the website's javascript. Only use it for public keys.
```suggestion
export const twak_property_id = process.env.TWAK_PROPERTY_ID;
```